sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusOptions.cs

```diff
@@ -193,19 +193,19 @@ internal Task ExceptionReceivedHandler(ProcessErrorEventArgs args)
             return Task.CompletedTask;
         }
 
-        internal ServiceBusProcessorOptions ToProcessorOptions() =>
+        internal ServiceBusProcessorOptions ToProcessorOptions(bool autoCompleteMessages) =>
             new ServiceBusProcessorOptions
             {
-                AutoCompleteMessages = AutoCompleteMessages,
+                AutoCompleteMessages = autoCompleteMessages,
```

Comment:
Shouldn't this be `autoCompleteMessages || AutoCompleteMessages`?

Reply:
Ah, I see that this happens in https://github.com/Azure/azure-sdk-for-net/pull/21354/files#diff-4eb12192433a31850c18d8e3df6fc5d7b99eea7abc5b59d31090844f20c7eddcR100